packages/vm/src/evm/eei.ts

```diff
@@ -380,6 +380,25 @@ export default class EEI {
     }
   }
 
+  /**
+   * Store 256-bit a value in memory to transient storage.
+   * @param key - Storage key
+   * @param value - Storage value
+   */
+  transientStorageStore(key: Buffer, value: Buffer): void {
+    // @ts-ignore
```

Comment:
Instead of using `ts-ignore`, I could also do something like:

```js
if (typeof this._state.putContractTransientStorage === 'function') {
  // call it
}
```

Reply:
sure, or another way we use frequently is the `in` operator: `if ('putContractTransientStorage' in this._state) {`

but here instead maybe throw an error if it is not available? it would add more safety for it being accidentally called

Reply:
I've updated the code to follow this style